src/darwin/Framework/CHIP/CHIPPersistentStorageDelegate.h

```diff
@@ -47,13 +37,13 @@ typedef void (^CHIPSendDeleteStatus)(NSString * key, NSError * status);
  * Set the value of the key to the given value
  *
  */
-- (void)CHIPSetKeyValue:(NSString *)key value:(NSString *)value handler:(CHIPSendSetStatus)completionHandler;
```

Comment:
this needs to be a a dispatch async callback on Darwin, doesn't match Darwin style otherwise